README.Rmd

```diff
@@ -323,7 +324,7 @@ events <- rbind(
 
 # Full join
 data <- full_join(data, events, by = c("date", "Package")) %>%
-  mutate(Package = fct_reorder(Package, count, .fun = sum, .desc = TRUE))
+  mutate(Package = as.character(fct_reorder(Package, count, .fun = sum, .desc = TRUE)))
```

Comment:
`fct_reorder()` is from _forcats_ package, right?

Reply:
yes it is. Btw I added `as.character` here because `poorman::full_join` fails otherwise, not sure why

Reply:
Maybe you can use `data_join()` function here from _datawizard_?

Also, any way to not use `fct_reorder()` and rely only on the base? That'll also help us get rid of _forcats_.

Reply:
I would say that we should do that later, in a second step. I'm not sure if this is just an one-liner or some more lines of code.